postgres/manage.sls

```diff
@@ -44,7 +44,25 @@ postgres-reload-modules:
 # Database states
 
 {%- for name, db in postgres.databases|dictsort() %}
-
+  {%- if 'extensions' in db %}
+    {%- for ext_name, extension in db.pop('extensions')|dictsort() %}
+      {%- do extension.update({'maintenance_db': name }) %}
+{{ format_state( name + '-' + ext_name, 'postgres_extension', extension) }}
```

Comment:
Would you mind to separate `format_state` macro call and related parameters from other Jinja flow control operators?
This would much improve the code readability.

Reply:
Maybe you consider to put newlines between state generation code and Jinja loops?